jenkins/build.sh

```diff
@@ -18,18 +18,59 @@ set -e
 
 PROG=$(basename "$0")
 SCRIPT_DIR=$(realpath $(dirname "$0"))
-BASE_DIR=$SCRIPT_DIR/..
+BASE_DIR="$SCRIPT_DIR"/..
 BUILD_NUMBER=
 GIT_VER=
 PRODUCT="cortx"
+ADDB_PLUGIN_DIR="$BASE_DIR"/src/addb_plugin
+MOTR_REPO="$BASE_DIR"/../cortx-motr
+RGW_REPO="$BASE_DIR"/../cortx-rgw
 
 usage() {
-    echo """usage: $PROG [-v version] [-g git_version] [-b build_number]""" 1>&2;
+    echo """usage: $PROG [-v version] [-g git_version] [-b build_number][-a]
+    where,
+          -v <version_id>   Specify rpm version
+          -g <git_version>  Specify git version hash
+          -b <build_number> Specify build version
+          -a                Generates addb plugin as part of cortx-rgw-integration""" 1>&2;
```

Comment:
Ttripple quotes are not required here, (bash script). /n might be needed. Please check once.

Reply:
done